AsyncDisplayKit/ASCellNode.h

```diff
@@ -167,6 +161,27 @@ typedef NS_ENUM(NSUInteger, ASCellNodeVisibilityEvent) {
  */
 - (void)cellNodeVisibilityEvent:(ASCellNodeVisibilityEvent)event inScrollView:(nullable UIScrollView *)scrollView withCellFrame:(CGRect)cellFrame;
 
+#pragma mark - UITableViewCell specific passthrough properties
+
+/* @abstract The selection style when a tap on a cell occurs
+ * @default UITableViewCellSelectionStyleDefault
+ * ASTableView uses these properties when configuring UITableViewCells that host ASCellNodes.
+ */
+@property (nonatomic) UITableViewCellSelectionStyle selectionStyle;
+
+
```

Comment:
I would say selectionStyle should be always have a default value. Its not necessary but it hints to users that its possible to configure other properties..

lines will be fixed.